test/command/suite/sharding/logical_count/log/range_index.test

```diff
@@ -6,6 +6,10 @@ table_create Logs_20150709 TABLE_NO_KEY
 column_create Logs_20150709 timestamp COLUMN_SCALAR Time
 table_create Times_20150709 TABLE_PAT_KEY Time
 column_create Times_20150709 timestamp_index COLUMN_INDEX Logs_20150709 timestamp
+load --table Logs_20150709
+[
+{"timestamp": "2015-02-05 13:49:00"}
+]
 
```

Comment:
I have added this because the expected log `#|d| [logical_count][range-index] <Logs_20150709>: range index is available` is not written if `Logs_20150709` is empty.

Currently, an empty table is skipped before logging that log.
https://github.com/groonga/groonga/pull/1341/files#diff-d3fb57a047ca92c9fc19d900aa1a906a31bc3519eb7c26927516b229705c27a4R290